blackbox/docs/sql/reference/create_table.txt

```diff
@@ -265,6 +265,18 @@ to 1000 milliseconds.
 
 For further details see :ref:`refresh_data` or :ref:`sql_ref_refresh`.
 
+.. _sql_ref_write_wait_for_active_shards:
+
+``write_wait_for_active_shards``
+................................
+
+  Specifies the number of active shard copies to check for before proceeding
```

Comment:
This only says that a "check" is happening, but it doesn't indicate what's happening based on this check or what's the purpose of the check.
I think something along the lines of: Specifies the number of shard copies that need to be active for the write operation to proceed.If less shard are active ...